lib/components/narrative/default/default-itinerary.js

```diff
@@ -192,3 +271,14 @@ export default class DefaultItinerary extends NarrativeItinerary {
     )
   }
 }
+
+const mapStateToProps = (state, ownProps) => {
+  return {
+    currency: state.otp.config.localization?.currency || 'USD',
+    locale: state.otp.ui.locale,
+    messages: state.otp.ui.messages,
```

Comment:
These are no longer used, please remove.

Reply:
Good catch, updated in https://github.com/opentripplanner/otp-react-redux/pull/375/commits/a9fdcf11538255d1d8ace74fd7fe1ee0fbd7cd64.